code/modules/vending/engivend.dm

```diff
@@ -18,7 +18,8 @@
 					/obj/item/electronics/advanced_airlock_controller = 10,
 					/obj/item/electronics/firealarm = 10,
 					/obj/item/electronics/firelock = 10,
-					/obj/item/storage/bag/construction = 3)
+					/obj/item/storage/bag/construction = 3,
+					/obj/item/storage/backpack/welding = 3)
```

Comment:
This feels way too makeshift to be an official nanotrasen supplied item, more like some kind of weird ghetto crafted item made with a tank and a backpack or something